pkgs/development/libraries/glib-testing/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, gtk-doc
+, docbook-xsl-nons
+, docbook_xml_dtd_43
+, glib
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libglib-testing";
```

Comment:
so with this `pname` will not equal the attribute name `glib-testing`.
Can we use `glib-testing` and not reuse it in `repo = pname;` below?